cogs/smallfunctions.py

```diff
@@ -0,0 +1,43 @@
+import discord
+import random
+import #new file storing priv variables
+from discord.ext import commands
+
+# Class responsible for all the small commands run by server members.
+class SmallFunctions(commands.cog):
+
+# Command function for randomized coinflip. Will return a heads or tails.
+    @commands.command()
+    async def random_coinflip(self, ctx):
+        heads = 1#store in diff file as const variables
+        tails = 0#store in diff file as const variables
+        rand_cf = random.randint(0,1)
+        if (rand_cf == heads):
+            return #gif for heads
+        else:
+            return #gif for tails
+
+
+# Command function decide which side you believe the coin will land on.
+    @commands.command()
+    async def decided_coinflip(self, ctx):
+        decided_cf = random.randint(0,1)
+
+
+# Command function that will be a help guide for RasenBot
+    @commands.command()
+    async def help(self, ctx):
```

Comment:
Would be cool if the help function sends a message that looks visually appealing like the greetings message. Does not have to be the same but should look good. Maybe try and look at other bots in how they format their help commands such as Mee6 or Carl

Reply:
I'll look into that